lib/Applify.pm

```diff
@@ -87,6 +87,9 @@ sub import {
 
   $self->{skip_subs} = {app => 1, option => 1, version => 1, documentation => 1, extends => 1, subcommand => 1};
 
+  $self->option_parser->configure(@{delete $args{getopt_config}})
+    if $args{getopt_config};
```

Comment:
I think we should merge this with the `option_parser()` attribute, and maybe make it read-only:

```
sub import {
  ...
  $args{getopt_config} ||= [qw(no_auto_help no_auto_version pass_through)];
  $self->option_parser->configure(@{delete $args{getopt_config}});
  ...
}
...
sub option_parser {
  return shift->{option_parser} ||= do { require Getopt::Long; Getopt::Long::Parser->new };
}
```